go-controller/pkg/ovn/pods.go

```diff
@@ -618,17 +617,19 @@ func (oc *Controller) getPortAddresses(nodeName, portName string) (net.HardwareA
 }
 
 // ovnNBLSPDel deletes the given logical switch using the libovsdb library
-func ovnNBLSPDel(client libovsdbclient.Client, logicalPort, logicalSwitch string) error {
+func (oc *Controller) ovnNBLSPDel(client libovsdbclient.Client, logicalPort, logicalSwitch string) error {
```

Comment:
@JacobTanenbaum now that this is a Controller method, we can just use `oc.nbClient` directly and we don't need to pass the client object into the method.

Reply:
done